index.css

```diff
@@ -622,3 +650,372 @@ footer {
   align-items: center;
   flex-wrap: wrap;
 }
+
+.desk-menu {
+  display: none;
+}
+
+.hidemb {
+  display: none;
+}
+
+.abimg {
+  display: none;
+}
+
+.meow {
+  background-color: hsla(22, 6%, 10%, 1);
+}
+
+@media (min-width: 768px) {
+  .mob-menu {
+    display: none;
+  }
+
+  body {
+    display: grid;
+  }
```

Comment:
- [ ] kindly fix the scroll bar horizontally which hides the design as appears on the following screen. I think the reason for that is because you give the body `display: grid;`, you can make a container and give it property.
> Your design
![image](https://user-images.githubusercontent.com/53402600/178014681-10aec318-39fc-409d-b0db-e83b3df791a7.png)